.travis.yml

```diff
@@ -18,17 +24,20 @@ before_install:
     - if [ ${TRAVIS_PYTHON_VERSION:0:1} == "3" ]; then export PY3="true"; else export PY2="true"; fi
     - if [ -n "${PY3}" ]; then export MONGO_VERSION=3.0.7; export PY_COVG="OFF"; else export MONGO_VERSION=2.6.11; export PY_COVG="ON"; export DEPLOY=true; fi
     - if [ -n "${PY3}" ]; then export IGNORE_PLUGINS=hdfs_assetstore,metadata_extractor; fi
-    - CACHE=$HOME/.cache source ./scripts/install_mongo.sh
+    - if [ "$GIRDER_TEST_GROUP" != "other" ] ; then CACHE=$HOME/.cache source ./scripts/install_mongo.sh ; fi
```

Comment:
I was originally trying to limit the number of things being installed into each container to speed up the setup time.  Unfortunately, we need basically a full setup in each one.  This is the only optimization that survived.  In reality, this step probably doesn't save any time.